tardis/energy_input/gamma_ray_packet_source.py

```diff
@@ -704,61 +738,46 @@ def create_packets(
         packet_energies_cmf = np.zeros(number_of_packets)
         nus_rf = np.zeros(number_of_packets)
         nus_cmf = np.zeros(number_of_packets)
-        times = np.zeros(number_of_packets)
-        # set packets to IN_PROCESS status
         statuses = np.ones(number_of_packets, dtype=np.int64) * 3
 
-        self.energy_plot_positron_rows = np.zeros((number_of_packets, 4))
-
-        # compute positronium continuum
-        positronium_energy, positronium_intensity = positronium_continuum()
+        # sample packets from the gamma-ray lines only (include X-rays!)
+        sampled_packets_df_gamma = decays_per_isotope[
+            decays_per_isotope["radiation"] == "g"
```

Comment:
This is a pretty weird operation, especially considering the documentation says that decays_per_isotope is an array of int64s. Is the documentation wrong?